.generated-sources/emily/openapi/build.rs

```diff
@@ -7,11 +7,26 @@ use utoipa::OpenApi;
 
 #[derive(utoipa::OpenApi)]
 #[openapi(
-    paths(api::handlers::check_address_handler,),
+    paths(
+        // Health check endpoints.
+        api::handlers::health::get_health,
+        // Deposit endpoints.
+        api::handlers::deposit::get_deposit,
+        api::handlers::deposit::get_deposits_for_transaction,
+        api::handlers::deposit::get_deposits,
+        api::handlers::deposit::create_deposit,
+        api::handlers::deposit::update_deposits,
+        // TODO: https://github.com/stacks-network/sbtc/issues/272
+        // Add withdrawal and chainstate endpoints.
```

Comment:
Great to see TODOs referencing issues. I typically format these a bit more concisely as

```suggestion
        // TODO(272): Add withdrawal and chainstate endpoints.
```